spec/services/prison_visits/api_spec.rb

```diff
@@ -135,10 +135,6 @@
   describe 'request_visit', vcr: { cassette_name: 'request_visit' } do
     subject { super().request_visit(valid_booking_params) }
 
-    before do
```

Comment:
Wouldn't checking that the names are in Welsh be a concern for pvb2? At a spec level the public app just cares that it passes the right information. That is how I was thinking about it.

What you describe sounds more like an integration test.